### ISSUE TITLE
Add a new Telemetry Uptake information about the percentage of the version with regards to all the pings

### ISSUE THREAD

**Issue by Natim** [mozilla/PollBot#172]
We want to be able to see what is the market share of the given version compared to all the version in use for the past 24 hours.

**Comment by Natim** [comment]
Something like that seems useful:

```
WITH updated_t AS (
    SELECT COUNT(*) AS updated
    FROM telemetry_update_parquet
    WHERE environment.build.build_id IN ('20180122144853')
      AND submission_date_s3 >= '2018012417'
),
total_t AS (
    SELECT COUNT(*) AS total, metadata.normalized_channel AS version
    FROM telemetry_update_parquet
    WHERE metadata.normalized_channel = 'aurora' AND submission_date_s3 >= '2018012417'
      GROUP BY 2
)
SELECT updated * 1.0 / total as ratio, updated, total, version
FROM updated_t, total_t
```

**Comment by Natim** [comment]
This doesn't work because the ping happens only once.

**Comment by Natim** [comment]
Maybe https://docs.telemetry.mozilla.org/datasets/batch_view/main_summary/reference.html#schema can help

**Comment by Natim** [comment]
```

WITH updated_t AS (
    SELECT COUNT(*) AS updated
    FROM main_summary
    WHERE submission_date_s3 >= '20180125'
      AND app_display_version = '58.0'
),
total_t AS (
    SELECT COUNT(*) AS total
    FROM main_summary
    WHERE submission_date_s3 >= '20180125'
      AND normalized_channel = 'release'
)
SELECT updated * 1.0 / total as ratio, updated, total
FROM updated_t, total_t
```

**Comment by Natim** [comment]
closed with #181
